airflow/dags/rt_views/gtfs_rt_fact_daily_feeds.sql

```diff
@@ -0,0 +1,62 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.gtfs_rt_fact_daily_feeds"
+
+description: |
+  Each row represents a GTFS realtime feed (combination of ITP ID, URL number,
+  and realtime data type) that was present in our extraction list on the given date.
+  Note that a feed can be added to the  list at any point in the day
+  and this table contains no information about extractions success or failure;
+  i.e., a feed may be listed here even if no data was successfully pulled,
+  or if only partial data was pulled.
+
+
+fields:
+  date: Date for which this feed was present in our extraction list
+  calitp_itp_id: Feed ITP ID
+  calitp_url_number: Feed URL number
+  url: Feed URL
+  type: GTFS realtime type (service_alerts, trip_updates, or vehicle_positions)
+
+tests:
+    check_null:
+        - calitp_itp_id
+        - calitp_url_number
+        - date
+        - type
+        - url
+    # do not include URL here in case we have duplicate labeling of different URLs
+    check_composite_unique:
+        - calitp_itp_id
+        - calitp_url_number
+        - date
+        - type
+
+dependencies:
+  - gtfs_schedule_fact_daily_feeds
+---
+
+-- note that for realtime we do not (yet) have a feed_key-type identifier
+-- so use calitp_itp_id plus calitp_url_number as identifier
+
+SELECT
+    calitp_itp_id,
+    calitp_url_number,
+    date,
```

Comment:
I'm not a huge fan of reserved keywords as column names; could this be `extraction_date` or similar?